layout/css/main.css

```diff
@@ -196,4 +208,12 @@ footer{
     -ms-transform: translateX(-50%);
     transform: translateX(-50%);
   }
+  .svgSpanner {
+	  height: 720px;
```

Comment:
Are we putting this specific height in an appropriate media query? If we are putting it in the general CSS this height is gonna look super odd on mobile.